samples/drivers/spi_bitbang/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	spibb0: spibb@0 {
+		label = "SPIBB";
+		compatible = "zephyr,spi-bitbang";
+		status="okay";
+		reg=<0 0x1000>;
```

Comment:
No need for `reg` or a unit address for these nodes, I think. You should be able to define these nodes without either if they're children of the root node as is being done here. See `/pwmleds` for example.

Reply:
I'm admittedly a bit of a fumbler for setting up device tree stuff, earlier code wouldn't work without the "reg" there and it just stayed :|   Fixed now.